nx.json

```diff
@@ -515,5 +515,14 @@
     "appsDir": "apps"
   },
   "useDaemonProcess": false,
-  "cacheDirectory": ".cache/nx"
+  "cacheDirectory": ".cache/nx",
+  "nxCloudId": "63e6951e043da20dd8321aec",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx-cloud",
+      "options": {
+        "nxCloudId": "63e6951e043da20dd8321aec"
```

Comment:
according to the docs, we should add `nx-cloud` devDependency for it to work on Nx <= 19.6 (https://github.com/nrwl/nx/issues/28414 :'( )

Reply:
Not sure to understand where in the doc it flags `nx-cloud` as mandatory devDep (and it is working locally today without it).
`nx-cloud` is convenient to register global on your machine but it is not mandatory in the dev in the repo (as the examples in the docs are `npx nx-cloud ...` based

Reply:
![image](https://github.com/user-attachments/assets/20e416bd-967e-4828-994b-30e10da45a85)

Reply:
Indeed :S
It's working without it today but let's add it then